src/applications/ezr/sass/_shame.scss

```diff
@@ -0,0 +1,5 @@
+.ezr-list-reset{
```

Comment:
You can eliminate this class and add a class name of `no-bullets` to the `<ul>`.

Reply:
Thank you! I was looking for that class